doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -145,6 +156,41 @@ The following properties are used if the particle impact one of the boundaries o
 
 * The ``youngs modulus`` parameter is the Young's modulus of the particle's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall.
 
+
+Box refinement
+---------------------
+For a particle to be accounted for in the fluid mesh, it has to overlap one or more vertices of this fluid mesh. If the initial mesh is too coarse in regards to the particle size, the particle may not be captured if it does not intersect the outer mesh walls.
+To avoid this, you can specify a region in the fluid domain where you want the mesh to be finer. To do so, a box refinement can be added with the following example parameters:
+
+.. code-block:: text
+
+	subsection  box refinement
+		subsection mesh
+			set type                 = dealii
+			set grid type            = subdivided_hyper_rectangle
+			set grid arguments       = 2,2,2: -1,-1,-1 : 1,1,1 : true
+			set initial refinement   = 0
+		end
+		set initial refinement   = 3
+	end
+
+* The ``mesh`` subsection allows to define the region in which the fluid mesh needs to be refined. A cell in the fluid mesh will be refined if at least one of its dofs is located within the outer boundaries of the box. Therefore, in this example, every cell of the fluid mesh that has at least one of its dofs located in the cube located between (-1, -1, -1) and (1,1,1) will be refined. For more information on meshes, see :doc:`../cfd/mesh`. 
+
+.. note::
+	The ``initial refinement`` of the ``subdivided_hyper_rectangle`` will not have any impact on the refinement of the fluid mesh, since only its shape and outer walls location are taken into account.
+
+* The ``initial refinement`` parameter in the ``box refinement`` subsection will dictate the number of times the fluid mesh cells inside the box will be refined. 
+
```

Comment:
This section should be moved to a separate section in general CFD parameters since it's a general tool for all solvers.